components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -8,9 +10,27 @@ import Select from "components/atoms/Select/select";
 import SelectOption from "components/atoms/Select/select-option";
 import LanguagePill from "components/atoms/LanguagePill/LanguagePill";
 
-const UserSettingsPage = () => {
+import { updateUser } from "lib/hooks/update-user";
+import { ToastTrigger } from "lib/utils/toast-trigger";
+import { useAuthSession } from "lib/hooks/useAuthSession";
+import { validateEmail } from "lib/utils/validate-email";
+import { timezones } from "lib/utils/timezones";
+
+interface userSettingsPageProps {
+  user: User | null;
+  sessionToken: string;
+}
+
+const UserSettingsPage = ({ user, sessionToken }: userSettingsPageProps) => {
+  const [isValidEmail, setIsValidEmail] = useState<boolean>(false);
+  const [email, setEmail] = useState<string | undefined>(user?.email);
   const [selectedInterest, setSelectedInterest] = useState<string[]>([]);
```

Comment:
I should have `interests` pre-selected from the data in my user profile, such as `javascript`